nnpdf_data/nnpdf_data/commondata/ATLAS_Z0_7TEV_49FB/metadata.yaml

```diff
@@ -1,61 +1,54 @@
 setname: ATLAS_Z0_7TEV_49FB
 version: 1
-version_comment: Port of old commondata
+version_comment: Reimplementation of old commondata
 nnpdf_metadata:
   nnpdf31_process: DY NC
   experiment: ATLAS
 arXiv:
   url: https://arxiv.org/abs/1305.4192
   journal: Phys.Lett. B725 (2013) 223
 iNSPIRE:
-  url: ''
+  url: 'https://inspirehep.net/literature/1234228'
 hepdata:
-  url: 10.17182/hepdata.61422.v1/t1
+  url: https://www.hepdata.net/record/ins1234228
   version: -1
 implemented_observables:
 - observable_name: HIMASS
   observable:
     description: Drell-Yan Mass Distribution
     label: ATLAS HM DY 7 TeV
     units: ''
-  process_type: EWK_MLL
-  tables: []
+  process_type: DY_MLL 
+  tables: [1]
   npoints: []
   ndata: 13
   plotting:
-    kinematics_override: ewk_mll_sqrt_scale
+    kinematics_override: identity
     dataset_label: ATLAS HM DY 7 TeV
     y_label: $d\sigma_{Z/\gamma^{*}}/dM_{ll}$ (fb)
     y_scale: log
-    plot_x: k2
+    plot_x: k2 #Should this be changed?
```

Comment:
Now that we have removed the mass squared kinematic, how should this line be changed such that the mass squared is still plotted?